### PR TITLE
Update Angular to version 14

### PR DESCRIPTION
**Pull request by matthew-morris-cessda**
See https://v14.angular.io/guide/update-to-latest-version for changes in Angular 14.

Most of the changes in this PR are caused by the introduction of typed `FormBuilder` instances and the migration of the codebase to use `UntypedFormBuilder` instances. Work to migrate to typed `FormBuilder` instances will take place in a separate PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,11 +29,11 @@
     "moment": "2.30.1",
     "ng-jhipster": "file:vendor/ng-jhipster-0.16.0.tgz",
     "ngx-chips": "2.2.2",
-    "ngx-cookie-service": "^13.0.0",
-    "ngx-infinite-scroll": "13.0.2",
-    "ngx-quill": "15.0.0",
+    "ngx-cookie-service": "14.0.1",
+    "ngx-infinite-scroll": "10.0.1",
```
Comment (alen-vodopijevec-cessda) [inline package.json:33]: Downgrade?

File: src/main/webapp/app/shared/tree-editor/tree-editor.component.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 -->
 <div *ngFor="let concept of conceptList">
     <ul *ngIf="concept.parent === parentNotation" class="concept-tree ul-level-{{level!}}">
-        <li [attr.id]="!concept.deprecated ? 'code_' + concept.notation?.split('.')?.join('') : null" [ngClass]="{'y-bgr': !concept.title}">
+        <li [attr.id]="!concept.deprecated ? 'code_' + concept.notation.split('.').join('') : null" [ngClass]="{'y-bgr': !concept.title}">
```
Comment (alen-vodopijevec-cessda) [inline src/main/webapp/app/shared/tree-editor/tree-editor.component.html:20]: Wouldn't this throw an error if 'concept.notation' is undefined?

**Review by alen-vodopijevec-cessda — COMMENTED** [review]
Check inline comment please.. just one thing to doublecheck.

File: package.json (hunk 1 above)
Comment (matthew-morris-cessda) [reply package.json:33]: Yes, and expected

File: src/main/webapp/app/shared/tree-editor/tree-editor.component.html (hunk 2 above)
Comment (matthew-morris-cessda) [reply src/main/webapp/app/shared/tree-editor/tree-editor.component.html:20]: `concept.notation` is defined to never be undefined in [src/main/webapp/app/shared/model/concept.model.ts](https://github.com/cessda/cessda.cvs.two/blob/651d4d39523e68e131ec6b4543ae67efbe9f753a/src/main/webapp/app/shared/model/concept.model.ts) so the check isn't necessary

**Review by alen-vodopijevec-cessda — APPROVED** [review]

**Pull request merged**